packages/rollups-subgraph/package.json

```diff
@@ -27,9 +27,7 @@
         "test": "graph test"
     },
     "dependencies": {
-        "@cartesi/rollups": "^0.6.0",
-        "@cartesi/rollups-0.8": "npm:@cartesi/rollups@0.8.2",
-        "@cartesi/rollups-0.9": "npm:@cartesi/rollups@0.9.0-alpha.2",
+        "@cartesi/rollups": "^0.9.0",
```

Comment:
I think that is fine because we (Cartesi) control these libraries, and they are well-aligned between the teams in terms of what is changing and if we need to make any big changes. e.g. create an alias because some contracts of interest will have a different address and we still want to index the previous one too.